src/main/java/roo/Roo.java

```diff
@@ -0,0 +1,152 @@
+package roo;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import roo.task.Deadline;
+import roo.task.Event;
+import roo.task.Todo;
+
+/**
+ * The main class for the Roo application, which is a tasks management program.
+ */
+public class Roo {
```

Comment:
Class names are all in PascalCase, variables and methods are in camelCase which is good!! Keep up the good work!!